#!/usr/bin/env python
PACKAGE = "distance_message"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("safeDistance", double_t, 0, "The safe distance parameter", 5, 0,   100)
gen.add("crashDistance", double_t, 0, "The crash distance parameter",1, 0,   100)



exit(gen.generate(PACKAGE, "distance_message", "dynamic_distance"))
